core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -149,7 +149,7 @@ class AdminClient(val time: Time,
       return List.empty[ConsumerSummary]
 
     if (group.protocolType != ConsumerProtocol.PROTOCOL_TYPE)
-      throw new IllegalArgumentException(s"Group ${groupId} with protocol type '${group.protocolType}' is not a valid consumer group")
+      throw new IllegalArgumentException(s"Group $groupId with protocol type '$group.protocolType' is not a valid consumer group")
```

Comment:
We need `{}` in `${group.protocolType}`, otherwise the behaviour is wrong. I would prefer if these string interpolation changes weren't included unless it's really obvious that it's fine as the compiler doesn't help us. For example, `$groupId` is obviously fine. But if there are quotes around the variable and so on, then I think it's fine to keep the braces.